folders.md

```
@@ -0,0 +1,1970 @@
+2. Inspects the faulting instruction
+3. If the opcode is `0x64` (`%fs` prefix), patches
+   it to `0x65` (`%gs` prefix) in-place
+4. Returns to retry the patched instruction
+
+If the fault is not TLS-related, it panics with the
+full register dump.
+
+### Running LibOS Mode
+
+**Via xtask:**
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)